ci/build_container/build_recipes/nghttp2.sh

```diff
@@ -7,11 +7,40 @@ set -e
 # TODO(PiotrSikora): switch back to releases once v1.33.0 is out.
 VERSION=e5b3f9addd49bca27e2f99c5c65a564eb5c0cf6d  # 2018-06-09
 
-wget -O nghttp2-"$VERSION".tar.gz https://github.com/nghttp2/nghttp2/archive/"$VERSION".tar.gz
+curl https://github.com/nghttp2/nghttp2/archive/"$VERSION".tar.gz -sLo nghttp2-"$VERSION".tar.gz
 tar xf nghttp2-"$VERSION".tar.gz
 cd nghttp2-"$VERSION"
-autoreconf -i
-automake
-autoconf
-./configure --prefix="$THIRDPARTY_BUILD" --enable-shared=no --enable-lib-only
-make V=1 install
+
+# Allow nghttp2 to build as static lib on Windows
+cat > nghttp2_cmakelists.diff << 'EOF'
+diff --git a/lib/CMakeLists.txt b/lib/CMakeLists.txt
+index 17e422b2..e58070f5 100644
+--- a/lib/CMakeLists.txt
++++ b/lib/CMakeLists.txt
+@@ -56,6 +56,7 @@ if(HAVE_CUNIT OR ENABLE_STATIC_LIB)
+     COMPILE_FLAGS "${WARNCFLAGS}"
+     VERSION ${LT_VERSION} SOVERSION ${LT_SOVERSION}
+     ARCHIVE_OUTPUT_NAME nghttp2
++    ARCHIVE_OUTPUT_DIRECTORY static
+   )
+   target_compile_definitions(nghttp2_static PUBLIC "-DNGHTTP2_STATICLIB")
+   if(ENABLE_STATIC_LIB)
+EOF
+
+if [[ "${OS}" == "Windows_NT" ]]; then
+  git apply nghttp2_cmakelists.diff
```

Comment:
Can you just use `patch` for this?

Reply:
`patch` doesn't come installed by default with MSYS on Windows